api.md

```diff
@@ -1874,6 +1860,44 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_continuous_aggregate_policy() :community_function> [](add_continuous_aggregate_policy)
+
+Create a policy that automatically refreshes a continuous aggregate.
+
+#### Required Arguments [](add_continuous_aggregate_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `continuous_aggregate` | (REGCLASS) The continuous aggregate to add the policy for. |
+| `start_interval` | (INTERVAL) Start of the refresh window relative to the time when the policy is executed. |
+| `end_interval` | (INTERVAL) End of the refresh window relative to the time when the policy is executed. |
+| `schedule_interval` | (INTERVAL) Interval between refresh executions. |
+
+#### Optional Arguments [](add_continuous_aggregate_policy-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `if_not_exists` | (BOOLEAN) Set to true to avoid throwing an error if the continuous aggregate policy already exists. A notice is issued instead. Defaults to false. |
+
+#### Returns [](add_continuous_aggregate_policy-returns)
+
+|Column|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id created to implement this policy|
+
+
+#### Sample Usage [](add_continuous_aggregate_policy-examples)
+
+To add a policy that refreshes the last month once an hour:
```

Comment:
```suggestion
Add a policy that refreshes the last month once an hour, excluding the latest hour from the aggregate (for performance reasons, it is recommended to exclude buckets that still see lots of writes):
```

The above can be complemented by linking to a deeper discussion on recommended settings.